src/response.ts

```diff
@@ -34,6 +34,16 @@ export const defaultResponse: Omit<MockedResponse, 'headers'> = {
   once: false,
 }
 
+const JSONBodyResponseTransformer = (res: MockedResponse): MockedResponse => {
+  if (res.body && res.headers.get('content-type')?.endsWith('json')) {
+    res.body = JSON.stringify(res.body)
+  }
+
+  return res
+}
+
+const postProcessTransformers = [JSONBodyResponseTransformer]
```

Comment:
Let's drop the encapsulation of a single transformer into an array :) I know the intention here, but on practice this will remain an array with a single member forever. We can apply `JSONBodyResponseTransformer` directly.

Reply:
Yeah, I was torn here knowing there is a premature abstraction lurking on the corner :). I will simplify it. We can always abstract it later.